src/processor/operator/persistent/node_batch_insert.cpp

```diff
@@ -111,7 +108,8 @@ void NodeBatchInsert::initLocalStateInternal(ResultSet* resultSet, ExecutionCont
 
     nodeLocalState->nodeGroup = NodeGroupFactory::createNodeGroup(ColumnDataFormat::REGULAR,
         nodeInfo->columnTypes, info->compressionEnabled);
-    nodeLocalState->columnState = std::move(state);
+    KU_ASSERT(resultSet->dataChunks[0]);
```

Comment:
Can u also assert that there is only one dataChunk in the resultSet? This should be an implicit assumption we hold for now.

Reply:
When i gdb'd the example case, if saw there were 2 datachunks in the resultSet. I'm not sure if that implicit assumption is being upheld currently...